### DISCUSSION TITLE
Running rblaunch.bat issue question

### DISCUSSION THREAD

**Discussion by auralplane** [TheKikGen/REBIRTH-NOCD#2] [General]
Hi there,

I'm trying your patch with Windows XP (32 bit) with Rebirth installed under \Program Files\Propellerhead\Rebirth RB-338 2.0\
This is on WinXP32 on an Acer Aspire One.

The output is an error stating that PatchLauncher32.exe is not a valid Win32 application.
CMD prompt states access denied.

**Comment by TheKikGen** [comment]
Hmmm, it runs on my win 11 64.  Don't know why you have an issue here.  Are you sure to use th 32 bits version of the patcher ?

**Reply by auralplane** [reply]
This wasn't the only app that came up complaining about an invalid Win32 application. Not sure what the checks are to cause that. I had this with one of the drive mounting tools as well. I did find an open source drive mapping tool that works under Winxp 32 bit.
Something to do with the compiling for the Win32 executable?

**Reply by auralplane** [reply]
I was running the batch file as per instructions. I'm assuming something checks for 64 or 32 bit?
